src/styles/_loops.scss

```diff
@@ -0,0 +1,5 @@
+@for $num from 1 through 6 {
+  .blocks__block--#{$num} {
+    background-color: rgb(255 - (42.5 * $num),0,0);
```

Comment:
Try not to write magic numbers. Numbers like 42.5 should be written in a variable